nbs/models.ipynb

```diff
@@ -4057,6 +4057,16 @@
     "        return out"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        if yip == 0.:
</pre><p>Please fix this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='195'/>

Reply:
fixed in [07f9aba](https://github.com/Nixtla/statsforecast/pull/294/commits/07f9aba0f795c94e90b13a9c44edaaae7f07eb26)